br_nfe/models/invoice_eletronic.py

```diff
@@ -717,14 +730,14 @@ def _prepare_eletronic_invoice_values(self):
             vencimento = fields.Datetime.from_string(dup.data_vencimento)
             duplicatas.append({
                 'nDup': dup.numero_duplicata,
-                'dVenc':  vencimento.strftime('%Y-%m-%d'),
+                'dVenc': vencimento.strftime('%Y-%m-%d'),
                 'vDup': "%.02f" % dup.valor
             })
         cobr = {
             'fat': {
                 'nFat': self.numero_fatura or '',
                 'vOrig': "%.02f" % (
-                    self.fatura_liquido + self.fatura_desconto),
+                        self.fatura_liquido + self.fatura_desconto),
```

Comment:
br_nfe/models/invoice_eletronic.py:740:25: E126 continuation line over-indented for hanging indent